nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -214,7 +214,7 @@ in
             ${pkg}/opt/healthchecks/manage.py collectstatic --no-input
             ${pkg}/opt/healthchecks/manage.py remove_stale_contenttypes --no-input
             ${pkg}/opt/healthchecks/manage.py compress
-          '';
+          '' + lib.optionalString (cfg.settings.DEBUG != "True") "${pkg}/opt/healthchecks/manage.py compress";
```

Comment:
Nope, it's not me -- this line needs to *replace* the line above it, not be added to it (`compress` is in there twice).

Reply:
Right, thanks for catching. Should be fixed now.